core/src/main/scala/cromwell/core/WorkflowSourceFiles.scala

```diff
@@ -1,9 +1,11 @@
 package cromwell.core
 
+import better.files.File
 import wdl4s.{WdlJson, WdlSource}
 
 /**
   * Represents the collection of source files that a user submits to run a workflow
   */
 final case class WorkflowSourceFiles(wdlSource: WdlSource, inputsJson: WdlJson,
-                                     workflowOptionsJson: WorkflowOptionsJson)
+                                     workflowOptionsJson: WorkflowOptionsJson,
+                                     importsFile: Option[File] = None)
```

Comment:
In situations where there's a case class w/ a single optional field I prefer to see them structured as an ADT sum type where the two implementations are such that one does not have an importsFile and the other has a nonOptional one. 

In situations where there are more than one option the code gets too messy to deal with and I give up :)